scripts/color-classes-generator.js

```diff
@@ -35,7 +35,7 @@ const generateBGVariants = (value, index) => {
  * @returns scss string
  */
 exports.generateColorUtilitityClasses = (colorToken) => {
-	let output = '@import "variables";\n@import "color-placeholder";\n';
+	let output = '@use "variables" as *;\n@use "color-placeholder" as *;\n';
```

Comment:
template literals might be easier to read here.

Reply:
fixed